cmd/cli/plugin/cluster/upgrade.go

```diff
@@ -115,18 +115,15 @@ func upgradeCluster(server *configapi.Server, clusterName string) error {
 		return err
 	}
 
-	tkrVersion := ""
-	if uc.tkrName != "" {
```

Comment:
Why do we remove this condition here?

Reply:
because `getValidTkrVersionFromTkrForUpgrade` should return highest available tkr if `tkrName` is not specified. And it should be called if `tkrName` is empty.